app/models/note.rb

```diff
@@ -0,0 +1,21 @@
+class Note < ActiveRecord::Base
+  include ::Translatable
+
+  # create_table :notes do |t|
+  #   t.references :notable, polymorphic: true, index: true
+  #   t.references :person, index: true
+  #   t.timestamps
+
+
+  belongs_to :notable, polymorphic: true
+  belongs_to :person
```

Comment:
Okay.  I flipped it back.  I also changed the generic 'person' relations on ProjectStep and ProjectLog to 'agent'  (I decided that I didn't like 'contact' there)
Additionally added the foreign key constraints.